Makefile

```diff
@@ -12,8 +12,11 @@ VERSION=$(shell git describe --tags --always --dirty)
 ifeq ($(DOCKERIZED),y)
 all:
 	docker build -t kube-spawn-build:$(DOCKER_TAG) -f Dockerfile.build .
+	mkdir -p /tmp/.cache
+	chown -R $(UID):$(UID) /tmp/.cache
 	docker run --rm -ti \
 		-v `pwd`:/go/src/github.com/kinvolk/kube-spawn:Z \
+		-v /tmp/.cache:/tmp/.cache:Z \
```

Comment:
How about standard `~/.cache/go-build/`? Or even output of `go env GOCACHE`?

Reply:
Good point. I've changed the host cache directory to `GOCACHE`.